### ISSUE TITLE
[Table] overflow in xs action cell has renders incorrect

### ISSUE THREAD

**Issue by bjornalm** [carbon-design-system/carbon-addons-iot-react#3111]
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

The issue described in  #3013 is appearing for row size xs

### To Reproduce

1. Go to https://next.carbon-addons-iot-react.com/?path=/story/1-watson-iot-table-statefultable--stateful-example-with-row-nesting-and-fixed-columns
2. Select 'xs'  in knob 'size'
3. Click an action overflow menu button

### Expected behavior
Overflow renders the same as for the other sizes 

### Environment/versions:

- OS: [e.g. MacOS, Windows]
- Browser: [e.g. chrome, safari]
- `carbon-addons-iot-react` version: [e.g. `v2.60.0`]

### Additional context

![image](https://user-images.githubusercontent.com/5167091/144933127-02d221a1-6001-4fac-914d-cd3da8bfaf66.png)
### Specific timeline issues / requests

Do you want this work within a specific time period? Is it related to an
upcoming release?

_NB: The core contributors will try to work with your timeline, but it's not
guaranteed. The earlier you make a request in advance of a desired delivery
date, the better!_

**Comment by JordanWSmith15** [comment]
I also see an issue with "sm"

**Comment by bjornalm** [comment]
> I also see an issue with "sm"

Yes, but I think that will be fixed as part of #3062 where I'm fixing parts of the OverflowMenu styling which is what caused me to notice this.
